docs/maintainers/README.md

```diff
@@ -98,15 +98,15 @@ In these cases:
 
 ## Becoming a Repository Maintainer
 
-Repository Maintainers currently consist entirely of Microsoft employees.
+Repository Maintainers currently consist mostly of Microsoft employees.
 It is expected that over time, regular trusted contributors to the PowerShell repository will be made Repository Maintainers.
-Eligibility is heavily dependent on the level of contribution and expertise: individuals who contribute in meaningful ways to the project will be recognized accordingly.
+Eligibility is heavily dependent on the level of contribution and expertise: individuals who contribute in meaningful ways consistently to the project will be recognized accordingly.
```

Comment:
what about, "...individuals who **consistently** contribute in meaningful way..." instead?